source/features/tag-changes-link.tsx

```diff
@@ -110,11 +110,12 @@ async function init(): Promise<void> {
 			'.list-style-none > .d-inline-block:last-child', // Link to source tarball under release tag
 		], container.element);
 		for (const lastLink of lastLinks) {
+			const currentTag = allTags[index].tag;
 			const compareLink = (
 				<a
 					className="Link--muted tooltipped tooltipped-n"
-					aria-label={'See changes since ' + decodeURIComponent(previousTag)}
```

Comment:
IMO this "See changes since" is really confusing, to me it sounds like comparing `previousTag` with `HEAD`.

This commit just make it more straightforward. I choose `currentTag` instead of "this tag" because the tooltip happens to cover part of the tag name, and then you have to figure it out what it refers to:

<img width="181" alt="Screen Shot 2021-11-02 at 02 53 38" src="https://user-images.githubusercontent.com/44045911/139725345-f80cb164-9e13-4aac-b47d-24b9de2f7e71.png">